.github/workflows/apidoc.yml

```diff
@@ -6,27 +6,38 @@ jobs:
   apidoc:
     runs-on: ubuntu-latest
     steps:
-      - uses: docker://ruby:latest
       - uses: Homebrew/actions/git-ssh@master
         with:
           git_user: BrewTestBot
           git_email: homebrew-test-bot@lists.sfconservancy.org
           key: ${{ secrets.RUBYDOC_DEPLOY_KEY }}
-      - run: |
-          # silence bundler complaining about being root
-          mkdir ~/.bundle
-          echo 'BUNDLE_SILENCE_ROOT_WARNING: "1"' > ~/.bundle/config
-          
+
+      - name: Set up Ruby
+        uses: actions/setup-ruby@v1
+        with:
+          version: '>=2.3'
+
+      - name: Set up repositories
+        run: |
           # clone rubydoc.brew.sh with SSH so we can push back
           git clone git@github.com:Homebrew/rubydoc.brew.sh
           cd rubydoc.brew.sh
 
           # clone latest Homebrew/brew
           git clone --depth=1 https://github.com/Homebrew/brew
 
-          # run rake to build documentation
+      - name: Install RubyGems
+        run: |
+          # silence bundler complaining about being root
+          mkdir ~/.bundle
+          echo 'BUNDLE_SILENCE_ROOT_WARNING: "1"' > ~/.bundle/config
```

Comment:
Is this still necessary now that it isn't running in a Docker container anymore?

Reply:
If that's why it was in the original script, then you're likely correct.